### PR TITLE
Day2

### PR DESCRIPTION
**Pull request by amandakoster**

### CODE REVIEW

File: aboutme/script.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+var name = prompt('what is your name?');
+alert('welcome' + name);
```
Comment (traceyr) [inline aboutme/script.js:4]: I would add a space between welcome and name!

File: aboutme/script.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+console.log('Welcome ' + name + '!');
+
+var answer2 = prompt('Do you think Amanda is a native Seattleite?');
+console.log(answer2);
+if (answer2.toLowerCase() === 'yes' || answer2.toLowerCase() === 'y') {
```
Comment (traceyr) [inline aboutme/script.js:9]: This is great. That way the user can answer using either y/n or yes/no!

File: aboutme/script.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+console.log(answer4);
+if (answer4.toLowerCase() === 'yes' || answer4.toLowerCase() === 'y') {
+  alert('Yes! Even though I\'m not native, I think umbrellas are a pain.');
+} else if (answer4.toLowerCase() === 'no' || answer4.toLowerCase() === 'n') {
+  alert('Sorry, Gortex is cool.');
+} else {
+  alert('C\'mon, just answer the question!');
+}
+
+var answer5 = prompt('Last question. Do I say sub or hogie??');
```
Comment (traceyr) [inline aboutme/script.js:37]: I love this question. It is supposed to be yes/no but I like how you handled this :)

File: aboutme/style.css [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+  background-color: transparent;
+  -webkit-text-decoration-skip: objects;
+}
+
+
+footer {
+  padding: 30px;
+  text-align: center;
+  font-family: 'Open Sans', sans-serif;
+  color: 	#C0C0C0;
```
Comment (traceyr) [inline aboutme/style.css:53]: I would delete the extra space between color: and you color code.

File: aboutme/style.css [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+  font-family: 'Raleway', sans-serif;
+  padding: 20px;
+  float: left;
+  width: 50%;
+  color:gray;
+  display: inline-block;
+}
+
+aside {
+  float: right;
```
Comment (traceyr) [inline aboutme/style.css:35]: This was an adventure to figure out :) But it looks great on your page. Remember: if you ever use float, use clear:both on the element that falls right after you want it to stop so it doesn't do crazy things (thanks footer :)

File: aboutme/style.css [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+  width: 30%;
+  padding: 10px;
+  border: 1px dotted red;
+  font-family: 'Space Mono', monospace;
+  margin-right: 30px;
+   }
+
+a {
+  background-color: transparent;
+  -webkit-text-decoration-skip: objects;
```
Comment (traceyr) [inline aboutme/style.css:45]: I am interested to see what this does! Google

File: aboutme/index.html [hunk 7]
```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="style.css">
+    <script type="text/javascript" src="script.js"></script>
```
Comment (traceyr) [inline aboutme/index.html:6]: Best practice is to put this at the bottom of our html page, right before the closing body tag(so line 26)

File: aboutme/index.html [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+
+        <section> Something was missing. I wanted to keep building things, though I wanted to influence functionality and user experience along with directing visual communication.
+        </section>
+
+        <section>My interest in code began when I really wanted something to be green instead of blue on a Wordpress site. I tweaked the CSS code, it worked, and this piqued my interest. A few years later I was managing a website for a hackathon project, wanted our site to have a cool favicon, and to figure it out myself. I found the code and directions on Wikipedia, entered the code, saved the HTML file and hit refresh. When the red favicon appeared I was stunned. It felt like I had performed magic and wanted more.
+        </section>
+
+        <footer>Come back in 4 weeks, this will be so rad!</footer>
+
+
```
Comment (traceyr) [inline aboutme/index.html:26]: You have an extra space here. Perfect for putting script tags!

File: aboutme/script.js (hunk 1 above)
Comment (amandakoster) [reply aboutme/script.js:4]: Done!

File: aboutme/style.css (hunk 4 above)
Comment (amandakoster) [reply aboutme/style.css:53]: Done!

File: aboutme/style.css (hunk 6 above)
Comment (amandakoster) [reply aboutme/style.css:45]: This was part of  'CSS normalize' script that I found somewhere. I believe it made the background transparent to clear out any browser default colors.

File: aboutme/index.html (hunk 7 above)
Comment (amandakoster) [reply aboutme/index.html:6]: Done!

File: aboutme/index.html (hunk 8 above)
Comment (amandakoster) [reply aboutme/index.html:26]: Fixed it!

**Comment by amandakoster** [comment]
Thanks for the comments, that helps!

**Pull request merged**